packages/roosterjs-content-model-dom/lib/formatHandlers/common/borderFormatHandler.ts

```diff
@@ -45,6 +48,24 @@ export const borderFormatHandler: FormatHandler<BorderFormat> = {
             if (value && width != defaultWidth) {
                 format[key] = value == 'none' ? '' : value;
             }
+
+            if (isElementOfType(element, 'th') || isElementOfType(element, 'td')) {
+                const borderValues = extractBorderValues(format[key]);
```

Comment:
suggest we do a string search first, for example, search for "--var" in the string and only when it exist, we call extractBorderValues and do color transformation.